src/lib/utils/hooks/SynapseAPI/entity/queryKeys.ts

```diff
@@ -0,0 +1,91 @@
+import { QueryClient } from 'react-query'
+import {
+  EntityBundleRequest,
+  EntityChildrenRequest,
+  QueryBundleRequest,
+} from '../../../synapseTypes'
+
+/**
+ * Co-location of react-query keys for queries related to Synapse entity data. Two objectives of this strategy are
+ * - minimize the risk of inappropriate cache key collisions
+ * - simplify clearing the approprate query caches when the entity data is updated
+ *
+ * For more information, see https://tkdodo.eu/blog/leveraging-the-query-function-context
+ */
+export const entityQueryKeys = {
+  /** Query key for all entities */
+  all: [{ objectType: 'entity' }],
+  // Data for an entity
+  entity: (id: string) => [{ ...entityQueryKeys.all[0], id: id }],
```

Comment:
...or invalidate all cache data for a specific entity with `queryClient.invalidateQueries(entityQueryKeys.entity(entityId))`